src/component/registry/bpmn-elements-registry.ts

```diff
@@ -199,6 +199,21 @@ export class BpmnElementsRegistry {
   addOverlays(bpmnElementId: string, overlays: Overlay | Overlay[]): void {
     this.mxGraphCellUpdater.addOverlays(bpmnElementId, overlays);
   }
+
+  /**
+   * Remove all overlays of a BPMN element.
+   *
+   * @example
+   * ```javascript
+   * //  all overlays of the BPMN element with id: activity_1
+   * bpmnVisualization.bpmnElementsRegistry.removeAllOverlay('activity_1');
+   * ```
+   *
+   * @param bpmnElementId The BPMN id of the element where to remove the overlays
+   */
+  removeAllOverlays(bpmnElementId: string): void {
```

Comment:
❓ could be just name the method `removeOverlays` to prepare #1178 ? to identify which overlays to remove, we will only have to pass an additional optional paramater without having to change the API or add a new method

Reply:
I think we could rename it when we do  #1178, because for now, we remove only all overlays, and I didn't think about how to implement it.

Reply:
That's true, we don't know how we are going to implement the removal of specific overlays (we may not implement it soon as well).
So ok to keep this method name but in the TSDoc, can we put an `@experimental` annotation and state that this method is subject to name change when will implement #1178? That way we already warn users of a possible breaking change

Reply:
done in ad12c6f